lib/pointing_party_web/channels/room_channel.ex

```diff
@@ -0,0 +1,18 @@
+defmodule PointingPartyWeb.RoomChannel do
+  use PointingPartyWeb, :channel
+  alias PointingPartyWeb.Presence
+
+  def join("room:lobby", _payload, socket) do
+    send(self(), :after_join)
+
+    {:ok, socket}
+  end
+
+  def handle_info(:after_join, socket) do
+    push(socket, "presence_state", Presence.list(socket))
+    {:ok, _} = Presence.track(socket, socket.assigns.username, %{
+      online_at: inspect(System.system_time(:second))
```

Comment:
Did you mean for the `inspect` to stay?

Reply:
Nope! This is just some code I mostly copied and pasted from the Phoenix docs, and the PR is still a WIP. I'm about to work on this some more, and I'll clean this up.